src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -91,11 +91,15 @@ void Graph::CreateGraph(const std::vector<NodePtr>& graphNodes,
     this->graphNodes = graphNodes;
     this->graphEdges = graphEdges;
 
+    std::size_t parameter_index = 0;
+    std::size_t result_index = 0;
     for (auto node : graphNodes) {
         if ("Parameter" == node->getTypeStr()) {
-            inputNodesMap[node->getName()] = node;
+            inputNodesMap[parameter_index] = node;
```

Comment:
If we use index, why not use std::vector or other similar STL container? If yes, is there any performance benefit?

Reply:
I don't think so. If here use vector, how do we use the index to find node? Compare with cpu node?

Reply:
std::vector can use operator[]

Reply:
As we discussed, this PR should good enough, let do the future similar investigation/optimization in the separated PRs.